k8-operator/packages/api/api.go

```diff
@@ -51,43 +31,6 @@ func CallGetServiceTokenDetailsV2(httpClient *resty.Client) (GetServiceTokenDeta
 	return tokenDetailsResponse, nil
 }
 
-func CallGetSecretsV3(httpClient *resty.Client, request GetEncryptedSecretsV3Request) (GetEncryptedSecretsV3Response, error) {
```

Comment:
Hmm so one issue we might have is if people update their clients without updating infisical-core

Reply:
Doesn't matter because infisical core supported raw a long time ago. I mean raw endpoint was introduced long time back right?